apps/total-typescript/src/ability/ability.ts

```diff
@@ -45,17 +46,15 @@ const canViewTutorial = ({user, subscriber, module}: ViewerAbilityInput) => {
   return contentIsTutorial && Boolean(viewer)
 }
 
-const canViewWorkshop = ({user, module, lesson}: ViewerAbilityInput) => {
+const canViewWorkshop = ({user, module}: ViewerAbilityInput) => {
   const contentIsWorkshop = module?.moduleType === 'workshop'
 
   const purchases = user?.purchases || []
   const userHasPurchaseWithAccess = Boolean(
     purchases.find((purchase) => purchase.bulkCouponId === null),
   )
 
-  const hasVideo = Boolean(lesson?.muxPlaybackId)
```

Comment:
I was scratching my head at why "has video" was needed here but I don't think it matters. We'd want a different state entirely for that situation I reckon.